File: ios/TanakaImageViewManager.m
//
//  TanakaImageViewManager.m
//  tanakaanime
//
//  Created by Tanaka Mazivanhanga on 3/16/21.
//

#import "React/RCTViewManager.h"
@interface RCT_EXTERN_MODULE(TanakaImageViewManager, RCTViewManager)
RCT_EXPORT_VIEW_PROPERTY(url, NSString)
RCT_EXPORT_VIEW_PROPERTY(cornerRadius, NSNumber)
@end

File: ios/TanakaImageViewManager.swift
//
//  TanakaImageViewManager.swift
//  tanakaanime
//
//  Created by Tanaka Mazivanhanga on 3/16/21.
//

import UIKit
@objc(TanakaImageViewManager)
class TanakaImageViewManager: RCTViewManager{
  override class func requiresMainQueueSetup() -> Bool {
    return true
  }
  
  override func view() -> UIView! {
    return TanakaImageView()
  }
}
